packages/types/src/index.ts

```diff
@@ -343,6 +341,13 @@ export type Leg = {
   fareProducts?: Array<FareProduct>;
 };
 
+type Price = {
```

Comment:
I am already adding this type in my fare related PR, and I'm using the name `Money` so that it matches what is used in the GraphQL API. That meant replacing the old `Money` in our Types package with the new one. Take a look: 
https://github.com/ibi-group/otp-ui/commit/d1025105774caa3e9e2ddfd25dc6367ac0e537d3#diff-e71f2026e90f1ed0c00f8711eb855f0a8810f33dbf1491f5da9c97cdd513e5b4L349

How do you want to reconcile this? If you want to implement the same change then we can resolve this in a merge conflict (unless Git figures it out?) in whichever PR gets merged later.

Reply:
Another idea: since you probably need this file to be changed in a separate PR anyway, let's add all of our types changes in to that one PR. In fact, I already have a branch that I was preparing that you can add your changes to: https://github.com/ibi-group/otp-ui/tree/gql-transition/fare-product-types

Reply:
Is this the same? I think TNC fares are using a separate data structure

Reply:
It is. I have a fix I think you'll like